src/Audio.md

```diff
@@ -0,0 +1,110 @@
+# Audio Overview
+
+Game Boy audio is sometimes called "8-bit", but this doesn't mean much: the term is used to designate audio similar to that produced by 8-bit consoles.
+
+A better description would be "chiptune": tunes produced by a chip.
```

Comment:
No! No! No! Please avoid the use of the word chiptune. It's one of the most controversial terms in the chiptune scene itself, when it comes to an exact definition. It can means anything from an Amiga module with short loopes, so called "chipped", samples (the origin of the term) to a purist view that it can only refer to music generated by hardware, to a more liberal view that it can mean music in a cetain style, allowing a song to be made with a Game Boy but not be "chiptune". In short it can mean anything and everything depending on who you ask, so in the end it means nothing.

Reply:
Do you have a better wording for that sentence, or should it be removed entirely?

Reply:
Not sure. I guess both the first two sentences could be omitted, but then again, I'm not sure how much context you need to explain, in particular for younger readers. (God, I feel old...) Maybe a fair assumption is that anyone reading the document has heard at least a few example of music/sound produced by the Game Boy in games?

> Game Boy audio is sometimes called "8-bit", but this doesn't mean much: the term is used to designate audio similar to that produced by 8-bit consoles.

As for this sentence. It's true that 8-bit can be used to designate audio similar to that produced by 8-bit consoles (similar to how chiptune can mean anything and everything) but that sentence almost seems to imply that "8-bit" is used only to describe _similar_ sounds, not the actual thing itself.

Maybe something like?

> Game Boy audio is sometimes called "8-bit". This does not refer to the bit depth of the sound generated, but rather that it has sound capabilities typical of 8-bit consoles. Like much of its contemporary hardware, the Game Boy produces sound generated by simple digital circuits.

Reply:
I like @nitro2k01 last suggestion.